.changeset/cyan-chefs-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/explorer": patch
+---
+
+Hosted SQL API integration has been added for Redstone and Garnet.
```

Comment:
Exploring worlds on Redstone and Garnet chains will now retrieve data from the hosted SQL indexer.